server/apps/publish/archive_publish_tests.py

```diff
@@ -349,7 +353,7 @@ def test_queue_transmission(self):
             self.assertEquals(0, queue_items.count())
```

Comment:
Not a part of this PR, but nevertheless - `assertEquals()` is deprecated. If you feel like it, you can replace all occurrences of it in the file with `assertEqual()`.